src/Microsoft.Sbom.Contracts/Contracts/SBOMValidationResult.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Sbom.Contracts
+{
+    /// <summary>
+    /// Represents the result of a SBOM validation action.
+    /// </summary>
+    public class SBOMValidationResult
```

Comment:
What if this was an abstract class and you created two classes that extend it like `SbomValidationSuccess` and `SbomValidationFailure`?  The first could effectively be empty as its existence implies success and the second could have just the errors.  This way you'd never risk erroneously have an instance that is both nominally successful as well as contains a non-empty list of errors.

Reply:
Fixed